### DISCUSSION TITLE
Object detection

### DISCUSSION THREAD

**Discussion by Leon83858** [Courseplay/courseplay#6813] [Q&A] [answered]
Make drivers detect rocks, trees, houses, fences etc near a field so tey dont collide and get stuck at/with them with a tool

For example: When you use a fertilizer and there is a rock the driver could try to rise the spray arms if that isnt working he could try another Turning maneuver

**Comment by Tensuko** [comment]
This is way more complicated then it looks like.
We try to stay on the field when the Pathfinder is used in turns (not pulled strayers).
Also you can't say that raise arms are always the best working solution.
You would have to check many different situations and give a correct solution for this.

Most easy solution: More headlands

**Comment by pvaiko** [comment, marked as answer]
You can also try enabling AI turns, that works for non-towed implements and has some logic to recover in case a vehicle gets stuck.
